### PR TITLE
Feature/#1 zoom out toolbar

### PR DESCRIPTION
**Pull request by deletidev**

### CODE REVIEW

File: src/core/providers/canvas-view-settings/canvas-view-settings.context.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  CanvasViewSettingsContextModel,
+  ceateInitialSettings
+} from './canvas-view-settings.model';
+
+export const CanvasViewSettingsContext =
+  React.createContext<CanvasViewSettingsContextModel>({
+    canvasViewSettings: ceateInitialSettings(),
+    zoomIn: () => {},
+    zoomOut: () => {}
+  });
```
Comment (brauliodiez) [inline src/core/providers/canvas-view-settings/canvas-view-settings.context.tsx:12]: You can set this to null

```diff
export const CanvasViewSettingsContext =
-  React.createContext<CanvasViewSettingsContextModel>({
+  React.createContext<CanvasViewSettingsContextModel | null>(
- {
-    canvasViewSettings: ceateInitialSettings(),
-    zoomIn: () => {},
-   zoomOut: () => {}
-
-  });
null);
```

The on the user Context add a try catch

File: src/core/providers/canvas-view-settings/canvas-view-settings.provider.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <CanvasViewSettingsContext.Provider
+      value={{ canvasViewSettings, zoomIn, zoomOut }}
+    >
+      {children}
+    </CanvasViewSettingsContext.Provider>
+  );
+};
+
+export const useCanvasViewSettingsContext = () =>
```
Comment (brauliodiez) [inline src/core/providers/canvas-view-settings/canvas-view-settings.provider.tsx:38]: If you are using null to initiaize the context (see comment above), you can do something like:

```diff
- export const useCanvasViewSettingsContext = ()  => { 
- export const useCanvasViewSettingsContext = () : CanvasViewSettingsContextModel => {
+  context = React.useContext(CanvasViewSettingsContext);
+  if(context === null) {
+   throw('useCanvasViewSettingsContext: looks like you have forgotten to add the provider on top of the app :)');
+  }
+
+ return context;
}
```

File: src/pods/canvas/canvas.pod.tsx [hunk 3]
```diff
@@ -1,38 +1,33 @@
...
-      </button>
       <div className={classes.container}>
         <svg
           xmlns="http://www.w3.org/2000/svg"
           className={classes.containerSvg}
           viewBox={`0 0 ${viewBoxSize.width} ${viewBoxSize.height}`} // Zoom play
-          width={size.width} // Explicit SVG canvas width TODO: configure
-          height={size.height} // Explicit SVG canvas height TODO: configure
+          width={canvasSize.width} // Explicit SVG canvas width TODO: configure
+          height={canvasSize.height} // Explicit SVG canvas height TODO: configure
```
Comment (brauliodiez) [inline src/pods/canvas/canvas.pod.tsx:30]: I think we can remove the comments

Comment (deletidev) [reply src/pods/canvas/canvas.pod.tsx]: It's fixed

File: src/core/providers/canvas-view-settings/canvas-view-settings.provider.tsx (hunk 2 above)
Comment (deletidev) [reply src/core/providers/canvas-view-settings/canvas-view-settings.provider.tsx]: It's fixed

File: src/core/providers/canvas-view-settings/canvas-view-settings.context.tsx (hunk 1 above)
Comment (deletidev) [reply src/core/providers/canvas-view-settings/canvas-view-settings.context.tsx]: It's fixed

**Pull request merged**